src/odinapi/views/read_apriori.py

```diff
@@ -31,17 +38,19 @@ def get_data(key):
     }
 
 
-def get_interpolation_weights(xs: np.array, x: float) -> Tuple[int, int, float, float]:
+def get_interpolation_weights(
+    xs: npt.NDArray, x: float
+) -> Tuple[int, int, float, float]:
     assert x > xs[0] and x <= xs[-1]
     ind2 = np.argmax(x <= xs)
     ind1 = ind2 - 1
     dx = xs[ind2] - xs[ind1]
     w1 = (xs[ind2] - x) / dx
-    return ind1, ind2, w1, 1.0 - w1
+    return int(ind1), int(ind2), w1, 1.0 - w1
 
 
 def get_interpolation_weights_for_lat(
-    lats: np.array, lat: float
+    lats: npt.NDArray, lat: float
```

Comment:
I think it is just as well to use `np.ndarray`. Unless you want to do fancy stuff, `npt.NDArray` is, I think, overkill. (`np.array` is wrong, however.)